tests/test_language.py

```diff
@@ -0,0 +1,66 @@
+import unittest
```

Comment:
This file is a little trivial, but it (afaik) executes all the main paths through the type-detection code, even though the result is the same.